### PR TITLE
#163391160 Implement handlers to fetch user's articles

### PR DESCRIPTION
**Pull request by rafmme**
#### What does this PR do?
This PR implements handlers for fetching articles of a user

#### Description of Task to be completed?
* Implement a handler to fetch one article of a user
* Implement a handler to fetch all articles of a user
* Implement filtering of the articles

#### How should this be manually tested?
* git clone the repository
* run npm install
* Use POSTMAN to make api request to `api/v1/user/articles`, `api/v1/user/articles/:slug`, `api/v1/user/articles?drafts`, `api/v1/user/articles?tags=example`, `api/v1/user/articles?published`

#### What are the relevant pivotal tracker stories?
[Get User's articles](https://www.pivotaltracker.com/story/show/163391160)

### CODE REVIEW

**Comment by tejiri4** [comment]
LGTM, yet

File: server/controllers/ArticleController.js [hunk 1]
```diff
@@ -118,7 +119,8 @@ class ArticleController {
       return response(
         res, 500, 'failure',
         'server error',
-        { message: 'Something went wrong on the server' }, null
+        { message: 'Something went wrong on the server' },
+        null
```
Comment (Bobsar0) [inline server/controllers/ArticleController.js]: No need for 'null' here

File: server/middlewares/AuthMiddleware.js [hunk 2]
```diff
@@ -48,6 +54,35 @@ class AuthMiddleware {
...
+    try {
+      const { userId } = req.user;
+      const user = await User.findOne({ where: { id: userId, isVerified: true } });
+      if (user) {
+        return next();
+      }
+      return response(
+        res, 403, 'failure', 'authorization error',
+        { message: 'Oops! Your account isn\'t verified yet.' },
+        null
```
Comment (Bobsar0) [inline server/middlewares/AuthMiddleware.js]: No need for null here

File: server/middlewares/AuthMiddleware.js [hunk 3]
```diff
@@ -48,6 +54,35 @@ class AuthMiddleware {
...
+  static async checkUserVerification(req, res, next) {
+    try {
+      const { userId } = req.user;
+      const user = await User.findOne({ where: { id: userId, isVerified: true } });
+      if (user) {
+        return next();
+      }
+      return response(
+        res, 403, 'failure', 'authorization error',
+        { message: 'Oops! Your account isn\'t verified yet.' },
```
Comment (Bobsar0) [inline server/middlewares/AuthMiddleware.js]: Why the need for the message object when the response function accepts message as input?

Comment (rafmme) [reply server/middlewares/AuthMiddleware.js]: The object is the message params for the function

**Comment by Sammuel09** [comment]
LGTM, yet.

**Pull request closed without merging**